kratos/utilities/cpp_tests_utilities.h

```diff
@@ -93,6 +98,12 @@ namespace CppTestsUtilities
         const bool Initialize = true
         );
 
+    /**
+     * @brief This method creates a pure (Element) simple geometry in 3D (tetrahedra)
+     * @param rModelPart Reference to the ModelPart containing the problem
+     */
+    void KRATOS_API(KRATOS_CORE) CreatePure3DGeometry(ModelPart& rModelPart);
```

Comment:
```suggestion
    void CreatePure3DGeometry(ModelPart& rModelPart);
```

Reply:
Does not compile in windows, it is a namespace. not a class